### DISCUSSION TITLE
x-transition.slide

### DISCUSSION THREAD

**Discussion by james0r** [alpinejs/alpine#2653] [3. Feature ideas]
Really the only reason I would reach for jQuery for years was to have access to its smooth slideUp(), slideDown() functions.

I've since found a little npm package that achieves this for me, but it would be a killer addition to Alpine to have some easy way to attach slideUp and slideDown effects.

Apologies if this is a thread somewhere else, I couldn't find it if it exists.

Thanks!

**Comment by visnaut** [comment]
Have you tried the [Collapse plug-in](https://alpinejs.dev/plugins/collapse)?

**Reply by james0r** [reply]
> Have you tried the [Collapse plug-in](https://alpinejs.dev/plugins/collapse)?

Looks very cool. Need to play with it. Still would love to see it in core. It's such a common effect.

**Reply by ekwoka** [reply]
That's why it's a Core plugin.

So you can have it if you want it.

> I would reach for jQuery for years was to have access to its smooth slideUp(), slideDown() functions.

So why not just extract the behavior to not require 45kb of legacy code?

**Reply by james0r** [reply]
Yeah this is super old. x-collapse working great. no complaints.

**Reply by ekwoka** [reply]
Oh, didn't notice. I got just the new notification of it closing
